languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -10918,6 +10918,10 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
 	                <token inflected="yes">können</token>
 	                <token skip="-1">mir</token>
 	                <token postag="VER.*" postag_regexp="yes"/>
+	            </antipattern>
+	            <antipattern>
+	                <token regexp="yes">ob|warum</token>
+	                <token skip="-1">mir</token>
```

Comment:
`skip="-1"` bedeutet, dass beliebig viele Tokens **nach** dem aktuellen Token übersprungen werden können. Da dein AP hier endet, bewirkt das skip hier nichts. Sollte es in das Token darüber?

Wenn du das fixt, kannst du den Branch gleich nochmal auf den aktuellen upstream/master rebasen? Dann geht der gefailte Test weg.